Makefile

```diff
@@ -1,4 +1,5 @@
 SHELL := /bin/bash
+KITT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
What do you think about setting KITT_DIR With KITT_DOMAIN?  Then this variable can be used in other git repos

Reply:
i just needed to figure out the current directory from within the Makefile to copy that dns config over to my local host.

Reply:
Then let’s leave KITT_DIR OUT SINCE make runs in the root of the kitt repo?  Can copy using relative paths, like cp ./etc/dns.conf